lore/encoders.py

```diff
@@ -595,6 +608,10 @@ def fit(self, data):
         else:
             self.categories = list(ids.id.unique())
 
+        if len(self.categories) <= 1:
```

Comment:
Nice sanity check. @abhayspawar has a nice lib that does this sort of check for all encoders. Would be nice to get it in :)